schemas/statsum-token-response.yml

```diff
@@ -26,9 +26,16 @@ properties:
     title:          BaseUrl
     description: |
       Base URL for the statsum server this project is allocated on.
+  rootUrl:
+    type:           string
+    format:         uri
+    title:          rootUrl
+    description: |
+      Root URL for a Taskcluster cluster.
```

Comment:
Two questions here:

 1. Will statsum be treated like all of the other services, even though it's in Go and not JS (in particular, will it be at `:rootUrl/api/v1/project/:project`?)
 1. If so, do we want to allow for that to run on a different rootUrl than the rest of the cluster?

My thinking is that if the answer to (1) is "yes", then we should define this response as always applying to the "statsum' service in the same instance of Taskcluster as the auth service -- so, no need to return a rootUrl or a baseUrl.  If the answer to (1) is "no", then perhaps we can rename baseUrl to something more specific like statsumUrl and define this response as returning a token valid at `:statsumUrl/v1/project/:project`.

I actually don't care either way what the answer to (1) is.  "Yes" would not be too difficult to implement, and would provide a nice starting place if we ever decide to start building more services in Golang.

Reply:
(1) is probably "no".

Reasoning being that we won't have load balancer in front of statsum (we can), but we wont have multiple instance behind the load balancer. If we need to load balance we would use a hash function to assign each `project` to a statsum server. If those statsum servers forward metrics to the same destination, that's no problem as long as they don't have the same project names.

If wanted statsum to be a TC service it should be accepting TC creds, if we want to do that, then it's a different service. In this case we should think of statsum the way we think of sentry.